keras_cv/layers/preprocessing3d/global_random_flipping_y.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing3d import base_augmentation_layer_3d
+from keras_cv.ops.point_cloud import wrap_angle_radians
+
+POINT_CLOUDS = base_augmentation_layer_3d.POINT_CLOUDS
+BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES
+
+
+class GlobalRandomFlippingY(base_augmentation_layer_3d.BaseAugmentationLayer3D):
+    """A preprocessing layer which flips point clouds and bounding boxes with respect to the Y axis during training.
+
+    This layer will flip the whole scene with respect to the Y axis.
+    During inference time, the output will be identical to input. Call the layer with `training=True` to flip the input.
+
+    Input shape:
+      point_clouds: 3D (multi frames) float32 Tensor with shape
+        [num of frames, num of points, num of point features].
+        The first 5 features are [x, y, z, class, range].
+      bounding_boxes: 3D (multi frames) float32 Tensor with shape
```

Comment:
@ianstenbit FYI, this would be a future improvement from our side to introduce 3d box format, given we already have 2d box format

Reply:
+1 let's discuss this when the input pipeline is ready.